### PR TITLE
Improvements to the repository.

### PR DESCRIPTION
**Pull request by shawnjb**

### CODE REVIEW

File: ProTips/Readme.md [hunk 1]
```diff
@@ -1,5 +0,0 @@
-# ProTips
```
Comment (asaxena0824) [inline ProTips/Readme.md:1]: I don't think we should remove this file, it makes editing the pro tips section easier :)

File: README.md [hunk 2]
```diff
@@ -1,57 +1,25 @@
-<p align="center">
```
Comment (asaxena0824) [inline README.md:1]: This section should not be removed :) I'd suggest making smaller changes as opposed to a big pr like this if you want it to be approved/accepted

File: README.md [hunk 3]
```diff
@@ -1,57 +1,25 @@
...
-* To make multiple functions work through a single prompt, remember to separate the instructions by a `delimiter`.
-* Do not overengineer the prompt, keep it simple but specific. 
-
-## Main Pro Tips
-
-* Use tick mark to indicate that the scripts are correct. By using your feedback as training data, we aim to create the single largest fine-tuned model for Roblox.
-* You can ask ScripterAI to add comments to the code by appending 'add comments to code' to your request.
-* If you are not satisfied with the results, you can keep clicking on 'Generate' again to regenerate the response until you get what you're looking for. Albeit, each click will be a considered a call.
-
-## Open Source
```
Comment (asaxena0824) [inline README.md:38]: SHould not be removed :)

File: README.md [hunk 4]
```diff
@@ -1,57 +1,25 @@
...
-* The repo will be always MIT licensed. 
-* Aside from our proprietary AI model, we will make code for every feature on our website public over the next several weeks. 
-
-## RPG Mode 
-* RPG Mode was launched on Jan 12th and it specializes in Simulator and NPC generation or related prompts. A multi-tab editor interface is made available to users to work simulataneously on functions under this mode. Here's a video [demo](https://www.youtube.com/watch?v=-MFZLoDBmaU) on RPG mode that compares normal responses vs those generated by RPG mode.
```
Comment (asaxena0824) [inline README.md:43]: Same for this and the section below

**Review by asaxena0824 — CHANGES_REQUESTED** [review]
You can either update this pr or make a new one, thanks for these changes either way :)

File: README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <p align="center">
   <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://github.com/asaxena0824/ScripterAI/blob/main/RSA_6.png?raw=true" height="140">
-    <img alt="logo" src="https://github.com/asaxena0824/ScripterAI/blob/main/RSA_6.png?raw=true" height="140">
+    <source media="(prefers-color-scheme: dark)" srcset="images/RSA_6.png" height="140">
+    <img alt="logo" src="images/RSA_6.png" height="140">
```
Comment (asaxena0824) [inline README.md:4]: Why this change? I think it's not showing the logo anymore :)

Comment (shawnjb) [reply README.md:4]: Relative paths should be used to reference files and images in a GitHub repository as they remain valid across different machines and operating systems, while absolute paths are specific to a file structure and may not work correctly when the repository is accessed from a different location.
